CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 All notable changes to this project will be documented in this file.
+## [1.3] - 2020-04-16
```

Comment:
nit: I'd argue this change isn't backwards compatible since the underlying functionality is changing but the interface is staying the same, and therefore would warrant a major version bump.

Thoughts?

Reply:
I considered this but I wasn't sure how to think about "backwards compatibility" for the `aggregate` gem. I suppose the fact that we are obliged to make changes to `elasticsearch_models` and `web` to make this work would be an argument for a major version bump.

Reply:
Done.